scripts/build-crossdock.sh

```diff
@@ -5,8 +5,23 @@ set -euxf -o pipefail
 BRANCH=${BRANCH:?'missing BRANCH env var'}
 COMMIT=${GITHUB_SHA::8}
 
+mode=${1-main}
+
 make build-and-run-crossdock
 
+if [[ "$mode" == "pr-only" ]]; then
```

Comment:
you are adding more steps, not changing how existing steps behave. Wouldn't the previous command still build everything?

Reply:
Ok, noted. Will incorporate the requested changes